### ISSUE TITLE
Mountstats rolls over with older kernels / high traffic

### ISSUE THREAD

**Issue by SuperQ** [prometheus/procfs#166]
A user running RHEL 7.x with Linux 3.10 reported mountstats collector returning negative values.

Snipit from `/proc/self/mountstats`:

```
device X.X.X.X:/export mounted on /target with fstype nfs4 statvers=1.1
...
        per-op statistics
              ACCESS: 2927395007 2927394995 0 526931094212 362996810236 18446743919241604546 1667369447 1953587717
             GETATTR: 3496133443 3496133439 0 601334946972 685241596164 18446743941975445342 2055274291 2376907228
              LOOKUP: 2440299007 2440299000 0 511100701900 135283040252 18446743979242238815 1334197895 1556584543
```

**Comment by SuperQ** [comment]
Created for tracking, this was fixed by #165.

**Comment by discordianfish** [comment]
Then this can be closed, right?
